packages/tinacms/src/toolkit/fields/plugins/date-field-plugin.tsx

```diff
@@ -1,22 +1,22 @@
-import * as React from 'react'
+import React from 'react'
 import { useEffect, useState, useRef } from 'react'
-import { InputProps, textFieldClasses } from '../components'
+import { type InputProps, textFieldClasses } from '../components'
 import { wrapFieldsWithMeta } from './wrap-field-with-meta'
 // we might be able to go back to react-datetime when https://github.com/arqex/react-datetime/pull/813 is merged
 import ReactDatetime from '../../react-datetime/DateTime'
 import type { DatetimepickerProps } from 'react-datetime'
 import { format, parse, DEFAULT_DATE_DISPLAY_FORMAT } from './date-format'
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore importing css is not recognized
-import { Field } from '../../forms'
+import type { Field } from '../../forms'
 
 export const DateField = wrapFieldsWithMeta<InputProps, DatetimepickerProps>(
   ({ input, field: { dateFormat, timeFormat, ...rest } }) => {
     return (
       <>
         <ReactDateTimeWithStyles
           value={input.value}
-          onFocus={input.onFocus}
+          // onFocus={input.onFocus}
```

Comment:
**question**: Why is this commented-out code retained?

Reply:
removed